include/deal.II/lac/read_write_vector.templates.h

```diff
@@ -556,7 +556,7 @@ namespace LinearAlgebra
           communication_pattern);
         AssertThrow(tpetra_comm_pattern != nullptr,
                     ExcMessage(
-                      std::string("The communication pattern is not of type ") +
+                      "The communication pattern is not of type " +
```

Comment:
```suggestion
                      "The communication pattern is not of type "
```
One of the testers is complaining here:
```
invalid operands of types ‘const char [42]’ and ‘const char [53]’ to binary ‘operator+’
```

Reply:
Ah, drats, I thought I had found all of these places. Should be fixed now.